IdentityCore/src/webview/response/MSIDSwitchBrowserResponse.m

```diff
@@ -74,6 +74,24 @@ - (BOOL)useV2WebResponseHandling
     return useV2WebResponseHandling;
 }
 
++ (BOOL)isDUNAActionUrl:(NSURL *)url operation:(NSString *)operation
+{
+    if (url == nil) return NO;
+    
+    NSArray *pathComponents = url.pathComponents;
+    if ([pathComponents count] < 2)
+    {
+        return NO;
+    }
+    
+    if ([pathComponents[1] isEqualToString:operation])
```

Comment:
possible to have upper lower case?

Reply:
we don't allow this, it has to be exact much.